base_import_odoo/views/import_odoo_database.xml

```diff
@@ -0,0 +1,155 @@
+                        attrs="{'invisible': [('cronjob_running', '=', False)]}"
+                    />
+                    <field name="cronjob_running" invisible="1" />
+                </header>
+                <sheet>
+                    <field
+                        name="status_html"
+                        attrs="{'invisible': [('status_html', '=', False)]}"
+                    />
+                    <group col="4" name="credentials">
+                        <field
+                            name="url"
+                            widget="url"
+                            attrs="{'readonly': [('cronjob_running', '=', True)]}"
+                        />
+                        <field
+                            name="database"
+                            attrs="{'readonly': [('cronjob_running', '=', True)]}"
+                        />
+                        <field
+                            name="user"
+                            attrs="{'readonly': [('cronjob_running', '=', True)]}"
+                        />
+                        <field
+                            name="password"
+                            password="True"
+                            attrs="{'readonly': [('cronjob_running', '=', True)], 'required': [('cronjob_running', '=', False)]}"
+                        />
+                        <field name="null_password" />
+                        <field name="duplicates" />
+                        <field
+                            name="cronjob_id"
+                            attrs="{'invisible': [('cronjob_id', '=', False)]}"
+                        />
+                    </group>
+                    <label for="import_line_ids" />
+                    <field
+                        name="import_line_ids"
+                        attrs="{'readonly': [('cronjob_running', '=', True)]}"
+                    >
+                        <tree editable="top">
+                            <field name="sequence" widget="handle" />
+                            <field name="model_id" />
+                            <field name="domain" />
+                            <field name="defaults" />
+                            <field
+                                name="postprocess"
+                                placeholder="# `vals` is the dictionary to be passed to create/write, and you have access to `env`, `_id`, `remote`"
+                            />
+                        </tree>
+                    </field>
+                    <label for="import_field_mappings" />
+                    <field
+                        name="import_field_mappings"
+                        attrs="{'readonly': [('cronjob_running', '=', True)]}"
+                    />
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <template id="view_import_odoo_database_qweb">
+        <script type="text/javascript">
+            odoo.define('base_import_odoo', function(require) {
```

Comment:
I haven't tested in 13.0, but in 14.0 in debug mode, I run into [Service base_import_odoo already defined](https://github.com/OCA/OCB/blob/14.0/addons/web/static/src/js/boot.js#L137). I think if you leave it empty, then it will just assign an [incremented job number](https://github.com/OCA/OCB/blob/14.0/addons/web/static/src/js/boot.js#L112).

Reply:
And, this should be changed to `__base_import_odoo__`.

For the above, [this refactor](https://github.com/OCA/server-tools/pull/2519/commits/37683a58f5fb85cd3db636f3a3e8a96c12884644) seems to have done the trick.

Reply:
thanks